com.unity.perception/Runtime/Randomization/Parameters/ParameterTypes/CategorialParameters/GameObjectParameter.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-namespace UnityEngine.Experimental.Perception.Randomization.Parameters
+namespace UnityEngine.Perception.Randomization.Parameters
```

Comment:
Since there are many users of these APIs now, we should add [MovedFrom] decorations to the public types so that the script updater will automatically update users' code when they update to the latest package. For more, see https://confluence.unity3d.com/display/DEV/Move+type+to+another+namespace

Reply:
Cool! I've added this attribute to all our public API's.